server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -742,6 +748,9 @@ private synchronized void replayLogs(long targetRevision) {
 
                 updateLastReplayedRevision(nextRevision);
                 info.lastReplayedRevision = nextRevision;
+                if (command instanceof UpdateServerStatusCommand) {
```

Comment:
this command is executed at line 734. Should we execute it twice?

Reply:
The command at 734L updates `StandaloneCommandExecutor` but this line updates `ZookeeperCommandExecutor`. We need to stop both executors because a command is executed by `StandaloneCommandExecutor` first and written to zk logs.
https://github.com/line/centraldogma/blob/27718d65b6555240d752c5e6909b43fd276b7f1c/server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java#L1126